### PR TITLE
Rollbar - gulp integration with sourcemaps

### PR DESCRIPTION
**Pull request by edoshor**
This is mostly related to build process of frontend.
We upload the generated source maps to Rollbar. 
We still need to find a way to inject the git SHA1 into the application config.code_version

### CODE REVIEW

File: frontend/config.json.sample [hunk 1]
```diff
@@ -52,5 +52,6 @@
   },
   "backendUri": "https://rt-dev.kbb1.com:5000/galaxy2/backend",
   "rollbarToken": "rollbar-access-token-comes-here",
+  "code_version": "git-sha1-deployed-should-be-here",
```
Comment (kkavod) [inline frontend/config.json.sample]: codeVersion

**Comment by kkavod** [comment]
I have changed the names to camelcase to fit JS coding style, do we need to do that for Rollbar vars too?

**Comment by iwuvjhdva** [comment]
Looks good except for the underscore notation

On Sun, Feb 21, 2016 at 9:53 AM, Kolya Kavod <notifications@github.com>
wrote:

> I have changed the names to camelcase to fit JS coding style, do we need
> to do that for Rollbar vars too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Bnei-Baruch/galaxy2/pull/20#issuecomment-186770348>.
>



-- 
Kind regards,
Konstantin Alexandrov

http://illuzia.net

**Comment by edoshor** [comment]
Underscore notation was to mimic the Rollbar style. I don't care...

**Pull request merged**